src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -135,6 +135,11 @@ function CustomFieldInputMultipleChoice(props) {
             {!props.readOnly && (
               <input
                 aria-labelledby={`${props.id}-label`}
+                aria-haspopup={'listbox'}
```

Comment:
BTW: you can use strings in JSX:
```suggestion
                aria-haspopup="listbox"
```